### ISSUE TITLE
[BUG][Python][Python-Aiohttp] util._deserialize throws AttributeError in Python 3.7

### ISSUE THREAD

**Issue by bejohu** [OpenAPITools/openapi-generator#2022]
##### Description

```
Traceback (most recent call last):
  File "*******/venv/lib64/python3.7/site-packages/aiohttp/web_protocol.py", line 418, in start
    resp = await task
  File "*******/venv/lib64/python3.7/site-packages/aiohttp/web_app.py", line 458, in _handle
    resp = await handler(request)
  File "*******/venv/lib64/python3.7/site-packages/aiohttp/web_middlewares.py", line 119, in impl
    return await handler(request)
  File "*******/venv/lib64/python3.7/site-packages/aiohttp/web_middlewares.py", line 119, in impl
    return await handler(request)
  File "*******/venv/lib64/python3.7/site-packages/connexion/apis/aiohttp_api.py", line 32, in oauth_problem_middleware
    response = yield from handler(request)
  File "*******/venv/lib64/python3.7/site-packages/connexion/decorators/coroutine_wrappers.py", line 23, in wrapper
    connexion_response = yield from connexion_response
  File "*******/venv/lib64/python3.7/site-packages/connexion/apis/aiohttp_api.py", line 198, in get_response
    response = yield from response
  File "*******/openapi_server/controllers/search_controller.py", line 22, in search
    search_results = SearchResults.from_dict(data)
  File "*******/openapi_server/models/search_results.py", line 56, in from_dict
    return util.deserialize_model(dikt, cls)
  File "*******/openapi_server/util.py", line 106, in deserialize_model
    setattr(instance, attr, _deserialize(value, attr_type))
  File "*******/openapi_server/util.py", line 29, in _deserialize
    elif type(klass) == typing.GenericMeta:
AttributeError: module 'typing' has no attribute 'GenericMeta'
```

* `typing.GenericMeta` is [not available](https://github.com/python/typing/issues/468) in Python 3.7
* Relying on `__extra__` is [not recommended](https://github.com/python/typing/issues/136#issuecomment-138174590)

##### openapi-generator version

4.0.0-SNAPSHOT

##### Suggested fix

In `_deserialize` in [util.mustache](modules/openapi-generator/src/main/resources/python-aiohttp/util.mustache) use [typing_inspect](https://pypi.org/project/typing-inspect/), package which [might get included](https://github.com/python/typing/issues/570) in [typing](https://docs.python.org/3/library/typing.html) anyway.

**Comment by GuillaumeSmaha** [comment]
Hi @bejohu,
Do you plan to create a PR for your commit https://github.com/bejohu/openapi-generator/commit/086580e9398cb23d6debb72dc629cd68c7931af4 ?

**Comment by wing328** [comment]
Closed with #2884 

@bejohu please pull the latest master to give it a try.
